results.tex

```diff
@@ -73,10 +73,13 @@ \subsection{Stochastic Jittering}
   \begin{subfigure}[b]{.5\columnwidth}
     \centering
     \includegraphics[width=\textwidth]{cactus_holland_jittering.png}
-    \caption{With jittering enabled.}
+    \caption{Jittering enabled.}
     \label{fig:jittering_with}
   \end{subfigure}
-  \caption{Stochastic jittering.}
+  \caption{Holland cactus dataset \emph{needCitation} exhibiting wood-grain
```

Comment:
I assumed it was, since I got it from kitware@bigmac:~/TestData3D.  Let me know, otherwise I will try to find a reference myself or just use different data.